docs/swarm/index.md

```diff
@@ -33,44 +33,45 @@ services. You don't need additional orchestration software to create or manage
 a Swarm.
 
 * **Decentralized design:** Instead of handling differentiation between node
-roles at deployment time, Swarm handles any specialization at runtime. You can
-deploy both kinds of nodes, managers and workers, using the Docker Engine.
-This means you can build an entire Swarm from a single disk image.
+roles at deployment time, the Docker Engine handles any specialization at
+runtime. You can deploy both kinds of nodes, managers and workers, using the
+Docker Engine. This means you can build an entire Swarm from a single disk
+image.
 
-* **Declarative service model:** Swarm uses a declarative syntax to let you
-define the desired state of the various services in your application stack.
-For example, you might describe an application comprised of a web front end
-service with message queueing services and a database backend.
+* **Declarative service model:** Docker Engine uses a declarative syntax to let
+you define the desired state of the various services in your application
+stack. For example, you might describe an application comprised of a web front
+end service with message queueing services and a database backend.
 
-* **Desired state reconciliation:** Swarm constantly monitors the cluster state
-and reconciles any differences between the actual state your expressed desired
-state.
+* **Desired state reconciliation:** The Swarm manager node constantly monitors
+the cluster state and reconcile any differences between the actual state your
+expressed desired state.
```

Comment:
Change "reconcile" to "reconciles".

I think it would be useful to show what this means in practice. "For example, if you set up a service to run 10 replicas of a container, and a worker machine hosting two of those replicas crashes, the manager will create two new replicas to replace the missing ones. These will be assigned to workers that are still up and running."